backend/nomad.go

```diff
@@ -137,3 +180,19 @@ func (n *Nomad) watchNodes() {
 		q = &api.QueryOptions{WaitIndex: waitIndex}
 	}
 }
+
+func (n *Nomad) watchMembers() {
+	for {
+		members, err := n.MembersWithID()
+		if err != nil {
+			log.Errorf("watch: unable to fetch members: %s", err)
+			time.Sleep(10 * time.Second)
+			continue
+		}
+
+		n.members = members
+		n.updateCh <- &Action{Type: fetchedMembers, Payload: members}
+
+		time.Sleep(1 * time.Second)
```

Comment:
yeah unfortunately there is no possibility to do a blocking query. One second might be a bit much though, are you ok with increasing this to five seconds?